package.json

```diff
@@ -29,7 +36,8 @@
     "firebase-tools": "^14.7.0",
     "husky": "^9.1.7",
     "prettier": "^3.5.3",
-    "tailwindcss": "^4",
+    "tailwindcss": "^4.1.10",
+    "tw-animate-css": "^1.3.4",
```

Comment:
Relying on `tw-animate-css` will be a problem down the line when you get to pages with more complex animations. Consider JavaScript animation libraries like [Motion](https://motion.dev) and [GSAP](https://gsap.com/). I personally find Motion very easy to use, much easier than CSS even!

Note that layout animations (called “Smart animate” in Figma) are **impossible to replicate with CSS**.